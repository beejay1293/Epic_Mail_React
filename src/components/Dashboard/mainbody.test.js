import React from 'react';
import { mount, shallow, render } from 'enzyme';
import Mainbody from './mainbody';
describe('Main Body', () => {
  it('should render correctly the main body', () => {
    const Main = shallow(<Mainbody />);
  
    expect(Main).toMatchSnapshot();
  });
});
