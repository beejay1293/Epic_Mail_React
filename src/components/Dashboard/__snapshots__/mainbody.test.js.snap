// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main Body should render correctly the main body 1`] = `
<div
  className="main__body"
>
  <TopContent />
  <MessageBody />
</div>
`;
